py3status/modules/volume_status.py

```diff
@@ -68,44 +71,35 @@
 import re
 import shlex
 
-from subprocess import check_output, call
+from subprocess import check_output, call, DEVNULL
 
+from abc import ABCMeta, abstractmethod
+from six import with_metaclass
 
-class Py3status:
-    """
-    """
-    # available configuration parameters
-    button_down = 0
-    button_mute = 0
-    button_up = 0
-    cache_timeout = 10
-    channel = 'Master'
-    device = 'default'
-    format = u'♪: {percentage}%'
-    format_muted = u'♪: muted'
-    threshold_bad = 20
-    threshold_degraded = 50
-    volume_delta = 5
 
-    # compares current volume to the thresholds, returns a color code
-    def _perc_to_color(self, string):
-        try:
-            value = int(string)
-        except ValueError:
-            return self.py3.COLOR_BAD
+class AudioBackend(with_metaclass(ABCMeta)):
```

Comment:
For one thing, it's overkill; (see http://stackoverflow.com/a/19328146/1795505)
For another, you've just pulled in a dependency on `six` that wasn't there before, for no measurable gain. Better to keep the dependencies low and accessibility high unless there's a strong use case.